src/es/components/controllers/withFacet/WithFacet.js

```diff
@@ -88,7 +88,7 @@ export default class WithFacet extends WebWorker() {
         currentRequestObj = Object.assign(currentRequestObj, { ppage: event.detail.ppage })
         const result = await this.webWorker(WithFacet.updateFilters, currentCompleteFilterObj, undefined, undefined)
         currentCompleteFilterObj = result[0]
-        currentRequestObj.filter = result[1]
+        currentRequestObj.filter = [...result[1], ...initialFilter.filter(filter => !result[1].find((resultFilterItem => resultFilterItem.id === filter.id)))]
```

Comment:
There is one bracket too many here.

```suggestion
        currentRequestObj.filter = [...result[1], ...initialFilter.filter(filter => !result[1].find(resultFilterItem => resultFilterItem.id === filter.id))]
```